grails-app/utils/au/org/emii/portal/HtmlCharacterEntityReferences.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2015 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
```

Comment:
My recommendation would be to:
 * Stuff those 2 functions in the same file
 * Have an apache 2.0 license header (optional)
 * State that this was deliberately taken from spring